tests/groupTree.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import sunbeam
+import datetimen
+
+spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+
+
+class TestGroupTree(unittest.TestCase):
+    spe3 = None
+
+    def setUp(self):
+        if self.spe3 is None:
+            self.spe3 = sunbeam.parse('spe3/SPE3CASE1.DATA')
+        time = es.schedule.start
```

Comment:
This does not look correct: The argument to the `schedule.groupTree( )` function should be an integer - the report step; whereas the `schedule.start` method returns a `time_t` value.